packages/graphql-cli-validate-magento-pwa-queries/lib/index.js

```diff
@@ -116,20 +120,23 @@ function getSupportedArguments(args) {
  * Creates a linter configuration with rules based on the current
  * clients and project.
  */
-function getValidator({ clients, project }) {
+function getValidator({ clients, project, schemaPath }) {
     const clientRules = clients.map(clientName => ({
         env: clientName,
-        projectName: project
+        projectName: project,
+        schemaJsonFilepath: schemaPath
     }));
 
-    const ruleDefinition = ['error', ...clientRules];
+    const ruleDefinition = ['warn', ...clientRules];
```

Comment:
This is debatable. I know some folks think warnings are code smells. If we want to leave this as `error` we should then fix all the issues in the current graphql files (see the screenshot in the PR).

Reply:
Yes, in this repo, we do prefer errors over warnings. That said, if you'd like to create an issue to fix all the warnings (and then flip it back from `warn` to `error`), that would be fine.